packages/land/contracts/interfaces/IErrors.sol

```diff
@@ -0,0 +1,24 @@
+//SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.23;
+
+/// @title Errors
+/// @author The Sandbox
+/// @notice Common errors
+abstract contract IErrors {
```

Comment:
Interface or abstract ?
IErrors

Reply:
OZ is doing it with a lib apparently
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Errors.sol

Reply:
or abstract but they don't put the "I"
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/c80b675b8db1d951b8b3734df59530d0d3be064b/contracts/utils/Nonces.sol#L8